Library/Formula/imagemagick.rb

```diff
@@ -1,9 +1,8 @@
 class Imagemagick < Formula
   desc "Tools and libraries to manipulate images in many formats"
   homepage "http://www.imagemagick.org"
-  url "http://www.imagemagick.org/download/releases/ImageMagick-6.9.1-4.tar.xz"
-  mirror "https://downloads.sourceforge.net/project/imagemagick/6.9.1-sources/ImageMagick-6.9.1-4.tar.xz"
-  sha256 "74973478058069ce44040ff76b61df713d70adcadfd207836cc0622cadf1e4bf"
+  url "http://www.imagemagick.org/download/releases/ImageMagick-6.9.1-5.tar.xz"
```

Comment:
Ah, sorry, I've made a typo, and not only that, the link is wrong anyway :( 
A .gz version is there, but I'm not sure that's much of a help:
https://fossies.org/linux/misc/ImageMagick-6.9.1-5.tar.gz